packages/remote-feature-flag-controller/src/remote-feature-flag-controller.test.ts

```diff
@@ -640,11 +645,154 @@ describe('RemoteFeatureFlagController', () => {
     it('should return default state', () => {
       expect(getDefaultRemoteFeatureFlagControllerState()).toStrictEqual({
         remoteFeatureFlags: {},
+        localOverrides: {},
+        rawRemoteFeatureFlags: {},
         cacheTimestamp: 0,
       });
     });
   });
 
+  describe('override functionality', () => {
+    describe('setFlagOverride', () => {
+      it('sets a local override for a feature flag', () => {
+        const controller = createController();
+
+        controller.setFlagOverride('testFlag', true);
+
+        expect(controller.state.localOverrides).toStrictEqual({
+          testFlag: true,
+        });
+      });
+
+      it('overwrites existing override for the same flag', () => {
+        const controller = createController();
+
+        controller.setFlagOverride('testFlag', true);
```

Comment:
Instead of calling the method to prepare the state, and then calling it again to test the intended behavior, what are your thoughts on passing initial state to `createControler`?

``` suggestion
        const controller = createController({
          options: {
            state: {
              localOverrides: {
                testFlag: true,
              },
            },
          },
        });